metricsbp/sys_stats.go

```diff
@@ -24,45 +28,49 @@ func pullRuntimeStats() (cpu cpuStats, mem runtime.MemStats) {
 // RunSysStats starts a goroutine to periodically pull and report sys stats.
 //
 // Canceling the context passed into NewStatsd will stop this goroutine.
-func (st *Statsd) RunSysStats(labels Labels) {
+func (st *Statsd) RunSysStats(_ Tags) {
```

Comment:
Oh I thought it was part of a config struct so I said leave it but ignore it to keep backward compatibility. Since this is a function arg, doing that is kinda defeating the point and we can just remove it. Also this is already a breaking change :)